locator/tablets.hh

```diff
@@ -348,6 +348,9 @@ public:
     /// \throws std::logic_error If the given id does not belong to this instance.
     dht::token_range get_token_range(tablet_id id) const;
 
+    /// Returns a vector of contiguous tokens where each pair describes a range of tablet.
+    future<std::vector<token>> get_flat_token_ranges() const;
```

Comment:
Why the `flat` in the name?

Reply:
This function returns `std::vector<token>` - not `std::vector<dht::token_range>`.

`flat` was intended to indicate, that his is 1D and ranges are described by the adjacent elements. 1D array is used by `dht::describe_ownership()`.

That was my intention

Reply:
Better to choose a new name, "flat" doesn't convey that.

Reply:
Replaced with `get_sorted_tokens()`